config/eslint.yaml

```diff
@@ -17,3 +17,4 @@ rules:
     # There are edge cases such as (x) => { foo: [] } which are syntactically
     # ambiguous, does it return an object or is that a labeled block?
     arrow-body-style: [off, always]
+    no-nested-ternary: off
```

Comment:
yeah, it was basically the cleanest thing I could do at this time. A lot of this will need to get changed into a dispatch system, but until then...